src/App.js

```diff
@@ -1,23 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
+import { Route, Routes } from 'react-router-dom';
+import './styles/App.module.css';
+import Header from './components/Header';
+import ArticleList from './components/ArticleList';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header />
+      <Routes>
+        <Route path="/api/articles" element={<ArticleList />} />
```

Comment:
/api/ is an endpoint convention in the back-end, but not something you'd expect in a front-end URL. Best to replace this with just /articles